### ISSUE TITLE
nil participant error when rendering tooltips

### ISSUE THREAD

**Issue by spMohanty** [AIcrowd/AIcrowd#1264]
The master branch currently gives the following error : 

```
undefined method `id' for nil:NilClass
```

```
<% 
    uniq_id = rand(1000)
    random_class = 'tooltip-user-wrapper-' + uniq_id.to_s + '-' + participant.id.to_s
%>
<div class="tooltip-user-wrapper <%= random_class %>">
    <div style="display: table-row;" data-container=".<%= random_class %>" data-html="true" data-toggle="tooltip" 

```

<img width="956" alt="Screenshot 2020-03-20 at 16 20 26" src="https://user-images.githubusercontent.com/1581312/77178362-43fe5900-6ac7-11ea-9d4d-5caeaa0bfd7e.png">

**Comment by spMohanty** [comment]
@piotrekpasciak : This is hapening because [here](https://github.com/AIcrowd/AIcrowd/blob/master/app/views/shared/discourse/_topics_list.html.erb#L29) we are not checking if the participant is actually present.

This is caused by the divergence between the list of participants on the rails-web-app and the discourse. Unlikely to happen on production, but will probably happen a lot during development.

**Comment by skbly7** [comment]
Fixed after #1264, #1267
